docs/getting_started.md

```diff
@@ -120,8 +120,8 @@ kubectl describe pod busybox-curl-7bd5767999-spf7v
     Host Port: 0/TCP
     Args:
       start
-      --uri
-      core.openfeature.dev/default/featureflagconfiguration-sample
+      --sources
+      - '[{"uri":"core.openfeature.dev/default/featureflagconfiguration-sample","provider":"kubernetes"}]'
     Environment:
       FLAGD_METRICS_PORT: 8014
```

Comment:
unrelated to these changes, but I believe that this is actually the default already, could potentially replace this with a non default value

Reply:
this is an example of `k describe pod <pod>` output. so yes, this is the default value :)